src/ethsign/ethsign.go

```diff
@@ -398,7 +398,7 @@ func main() {
 					} else if txtype == types.DynamicFeeTxType {
 						tx = types.NewTx(&types.DynamicFeeTx{
 							Nonce:     nonce,
-							To:        &common.Address{},
+							To:        nil,
```

Comment:
curious what the difference is

Reply:
Based on @hexonaut err report I'm guessing common.Address{} is the zero address, which is not the same as not having an address which is needed for creates. There was a comment in geth that led me to believe this is the rught way to do it, and it seems to work in the test

Reply:
ah ok yeah. I can also see this reading through the signer code in geth (e.g. [here](https://github.com/ethereum/go-ethereum/blob/401354976bb44f0ad4455ca1e0b5c0dc31d9a5f5/signer/core/apitypes/types.go#L103).